[INFO   ] Logger: Record log in /storage/emulated/0/organizer/.kivy/logs/kivy_20-06-05_16.txt
[INFO   ] Kivy: v1.11.0.dev0, git-8b6af30, 20190320
[INFO   ] Python: v3.7.2 (default, Mar 20 2019, 15:02:54) 
[GCC 8.2.0]
[INFO   ] KivyMD: v0.104.1
[INFO   ] Factory: 184 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil, img_gif (img_ffpyplayer ignored)
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <sdl2>
[INFO   ] GL: OpenGL version <b'OpenGL ES 3.2 V@331.0 (GIT@50f1b2d, I1b05bd271c) (Date:04/09/19)'>
[INFO   ] GL: OpenGL vendor <b'Qualcomm'>
[INFO   ] GL: OpenGL renderer <b'Adreno (TM) 505'>
[INFO   ] GL: OpenGL parsed version: 3, 2
[INFO   ] GL: Texture max size <16384>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] Text: Provider: sdl2
[INFO   ] GL: NPOT texture support is available
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/organizer/new_app.py", line 312, in <module>
[WARNING] stderr:     NewApp().run()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.7/site-packages/kivy/app.py", line 799, in run
[WARNING] stderr:     self.load_kv(filename=self.kv_file)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.7/site-packages/kivy/app.py", line 596, in load_kv
[WARNING] stderr:     root = Builder.load_file(rfilename)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.7/site-packages/kivy/lang/builder.py", line 301, in load_file
[WARNING] stderr:     return self.load_string(data, **kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.7/site-packages/kivy/lang/builder.py", line 368, in load_string
[WARNING] stderr:     parser = Parser(content=string, filename=fn)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.7/site-packages/kivy/lang/parser.py", line 392, in __init__
[WARNING] stderr:     self.parse(content)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.7/site-packages/kivy/lang/parser.py", line 501, in parse
[WARNING] stderr:     objects, remaining_lines = self.parse_level(0, lines)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.7/site-packages/kivy/lang/parser.py", line 605, in parse_level
[WARNING] stderr:     level + 1, lines[i:], spaces)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.7/site-packages/kivy/lang/parser.py", line 664, in parse_level
[WARNING] stderr:     if current_property[:3] == 'on_':
[WARNING] stderr: TypeError: 'NoneType' object is not subscriptable
